src/dialects/abstract/query-interface.js

```diff
@@ -330,7 +330,8 @@ export class QueryInterface {
       type: QueryTypes.SHOWTABLES,
     };
 
-    const showTablesSql = this.queryGenerator.showTablesQuery(this.sequelize.config.database);
+    // Snowflake is the only dialect that expects database name
```

Comment:
I just discovered this when looking into MariaDb -- don't know how I missed that 🤷.  I think an object should be passed in with named keys, instead of ordinal arguments to accommodate the various platforms, which might make it more flexible.  Maybe that's a bad pattern.